1.0/docs/start/first-element/step-3.md

```diff
@@ -0,0 +1,167 @@
+
+#### Example—setting the icon using markup
+{: .caption }
+
+
+<pre class="prettyprint">
+&lt;icon-toggle toggle-icon="favorite">&lt;/icon-toggle>
+</pre>
+
+
+#### Example—setting the icon from JavaScript
+{: .caption }
+
+
+<pre class="prettyprint">
+var myToggle = document.querySelector('icon-toggle');
+myToggle.toggleIcon = "favorite";
+</pre>
+
+
+**Learn more: attribute and property names.** You'll note that the markup example
+uses `toggle-icon`, not `toggleIcon`. Polymer represents camelCase property names
+using dash-case attribute names. To learn more, see <a href="https://www.polymer-project.org/1.0/docs/devguide/properties.html#property-name-mapping">Property
+name to attribute name mapping</a> in the Polymer library docs.
+{: .alert .alert-info }
+
+Next, add a declaration for the `toggleIcon` property.
+
+Find the script tag and add the following `properties` object to the element's prototype:
+
+#### icon-toggle.html
+{: .caption }
+
+
+<pre class="prettyprint">
+&lt;script>
+  Polymer({
+    /* this is the element's prototype */
+    is: 'icon-toggle',
+    properties: {
+      toggleIcon: String
+    }
+  });
+&lt;/script>
+&lt;/dom-module>
+</pre>
+
+Key information:
+
+  * Declaring a property in the `properties` object is a good idea if the property is going to be part of your element's
+public API.
+  * A simple property declaration like this one just includes the type (in this
+case, `String`).
+
+
+
+**Learn more: deserializing attributes.** The declared property type affects how Polymer converts, or <em>deserializes</em>
+the attribute value (always a string value) into a JavaScript property value.
+The default is `String`, so the declaration of `toggleIcon` is a formality here.
+To learn more, see <a href="https://www.polymer-project.org/1.0/docs/devguide/properties.html#attribute-deserialization">Attribute
+deserialization</a> in the Polymer docs.
+{: .alert .alert-info }
+
+The `properties` object also supports several more features. Add the following lines to add
+support for the `pressed` property:
+
+#### icon-toggle.html
+{: .caption }
+
+
+<pre class="prettyprint">
+  Polymer({
+    /* this is the element's prototype */
+    is: 'icon-toggle',
+    properties: {
+      toggleIcon: String,
+      pressed: {
+        type: Boolean,
+        value: false,
+        notify: true,
+        reflectToAttribute: true
+      }
+    }
+  });
+</pre>
+
+Key information:
+
+ *   For this more complicated property, you supply a configuration object with
+several fields.
+*   The `value` specifies the property's [default value](https://www.polymer-project.org/1.0/docs/devguide/properties.html#configure-values).
+*   The `notify` property tells Polymer to <em>generate property change events
+    </em>when the property value changes. This lets the change be observed by
```

Comment:
hm maybe the wording could be tweaked slightly. Instead of `generate property change events` would it be possible to change it to `dispatch property change events`?